frontend/src/utils/TokenService.js

```
@@ -18,53 +18,79 @@ const TokenService = {
    */
   isRateLimited() {
-    const lastSetTime = localStorage.getItem(this.lastTokenSetTimeKey);
-    if (!lastSetTime) return false;
-    
-    const timeSinceLastSet = Date.now() - parseInt(lastSetTime, 10);
-    return timeSinceLastSet < this.tokenRateLimitMs;
+    try {
+      const lastSetTime = localStorage.getItem(this.lastTokenSetTimeKey);
+      if (!lastSetTime) return false;
+      
+      const timeSinceLastSet = Date.now() - parseInt(lastSetTime, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)